src/vertification.ts

```diff
@@ -245,9 +277,11 @@ export async function handleVerification(message: Message) {
                 message.reply({ content: "Failed to send email address. Make sure your email address is valid." });
                 return;
             });
-        await message.reply({ content: "Please **DM the bot** with a 7 digit code sent to the email address." });
+        await message.reply({ content: "Please **DM the bot** with a 7 digit code sent to the email address. Type `cancel` if you wish to cancel the verification code." });
     } else if (!user_row) {
         await message.reply({ content: `Your email is not registered. You have not submitted your application to the [form](<${FORM_LINK}>).` });
```

Comment:
Rephrase it to say, you _may_ have not submitted, or your application _may_ have not been processed yet.